dev/system-test/firestore.ts

```diff
@@ -2972,91 +3001,100 @@ describe('BulkWriter class', () => {
 });
 
 describe('Client initialization', () => {
-  const ops: Array<[string, (coll: CollectionReference) => Promise<unknown>]> =
+  const ops: Array<
     [
-      ['CollectionReference.get()', randomColl => randomColl.get()],
-      ['CollectionReference.add()', randomColl => randomColl.add({})],
-      [
-        'CollectionReference.stream()',
-        randomColl => {
-          const deferred = new Deferred<void>();
-          randomColl.stream().on('finish', () => {
-            deferred.resolve();
-          });
-          return deferred.promise;
-        },
-      ],
-      [
-        'CollectionReference.listDocuments()',
-        randomColl => randomColl.listDocuments(),
-      ],
-      [
-        'CollectionReference.onSnapshot()',
-        randomColl => {
-          const deferred = new Deferred<void>();
-          const unsubscribe = randomColl.onSnapshot(() => {
-            unsubscribe();
-            deferred.resolve();
-          });
-          return deferred.promise;
-        },
-      ],
-      ['DocumentReference.get()', randomColl => randomColl.doc().get()],
-      ['DocumentReference.create()', randomColl => randomColl.doc().create({})],
-      ['DocumentReference.set()', randomColl => randomColl.doc().set({})],
-      [
-        'DocumentReference.update()',
-        async randomColl => {
-          const update = randomColl.doc().update('foo', 'bar');
+      string,
+      (coll: CollectionReference) => Promise<unknown>,
+      /* skip */ boolean?
+    ]
```

Comment:
Optional: Consider using an `interface` instead of a tuple, to improve readability. It has become a little unwieldy and using a "normal object" instead of a tuple could improve readability.

Reply:
I hate to say no to an idea like this, it would improve readability. The impact is very limited, only to this test, and I'm just trying to keep focused on the goal of running tests against the emulator.